packages/protobuf/src/registry.ts

```diff
@@ -52,6 +54,11 @@ export const typeRegistry: IMessageTypeRegistry = createRegistry(
   SctService,
   ShieldedPoolService,
   StakeService,
+
+  // Types not explicitly referenced by any above services should be added here.
+  // Otherwise, it will not be possible to serialize/deserialize these types if,
+  // e.g., they're used in an `Any` protobuf.
+  DutchAuction,
```

Comment:
It seems that this registry walks through the entire tree of Protobuf types, starting with services, then RPC methods, then data types served by those methods.

But `ViewService#auctions` doesn't actually reference `DutchAuction` — it just references `Any`, since multiple auction types can be served by that endpoint. So I have to manually add `DutchAuction` to the registry or else the registry won't know about it.

Reply:
Good call, done